crates/graphql/src/schema/objects/nft.rs

```diff
@@ -521,70 +571,514 @@ If no value is provided, it will return XSmall")))]
     }
 }
 
-#[derive(Debug, Clone)]
-pub struct CollectionNft(Nft);
+#[graphql_object(impl = NftExtValue, Context = AppContext)]
+impl Nft {
```

Comment:
Is the double imp needed?

Reply:
See code comments and PR description. In short: having two (nearly) identical `impl` blocks for `Nft` and `CollectionNft` was required so Apollo could pick up the available fields when generating client code